titiler/api/endpoints/cog.py

```diff
@@ -101,7 +101,7 @@ async def cog_metadata(
     return info
 
 
-params: Dict[str, Any] = {
+tile_response_codes: Dict[str, Any] = {
```

Comment:
Maybe we can moves this in https://github.com/developmentseed/titiler/blob/master/titiler/ressources/responses.py or https://github.com/developmentseed/titiler/blob/master/titiler/ressources/common.py

Reply:
Yes I agree it should be shared.  Although maybe having a `models/openapi.py` file or equivalent (maybe that lives in ressources?) makes the most sense here.  I would think that a lot of the openapi stuff is shared across routes.